Snowflake.Data.Tests/IntegrationTests/ConnectionMultiplePoolsIT.cs

```diff
@@ -279,7 +281,7 @@ public void TestConnectionPoolDisable()
         [Test]
         public void TestNewConnectionPoolClean()
         {
-            var connectionString = ConnectionString + "maxPoolSize=2;";
+            var connectionString = ConnectionString + "maxPoolSize=2;minPoolSize=1;";
```

Comment:
if I would remove minPoolSize it would be used the default value of 2 and then it won't be correct `Assert.AreEqual(1, SnowflakeDbConnectionPool.GetPool(conn1.ConnectionString).GetCurrentPoolSize());`